deployment/src/strongmind_deployment/autotag.py

```diff
@@ -3,50 +3,98 @@
 from typing import Dict
 from strongmind_deployment.taggable import is_taggable
 import pulumi
-
-# Usage:
-# config = pulumi.Config()
-# register_auto_tags({
-#     'user:Project': pulumi.get_project(),
-#     'user:Stack': pulumi.get_stack(),
-#     'user:Cost Center': config.require('costCenter'),
-# })
-#  Or to take existing defaults:
-# add_standard_billing_tags();
-# 
-# Or to take existing defaults with a couple of additional tags:
-# add_standard_billing_tags({
-#     'Customer': 'Acme Co',
-#     'Service': 'Foobar',
-# })
+import subprocess
 
 ########################
 ###### Entrypoint ######
 
-def get_standard_tags(extra_tags=dict):
+class StandardTags:
+    def __init__(
+        self,
+        extra_tags: dict,
+        project: str = None,
+        environment: str = None,
+        service: str = None,
+        product: str = None,
+        repository: str = None,
+    ):
```

Comment:
@markng @salty-opsgenie 
These tags seem appropriate.   I think I'm still missing the value in having the pulumi project name set to all the tag keys, as in container.py.
If we need to change any of this - I'll gladly do it!